pkg/kubelet/lifecycle/handlers_test.go

```diff
@@ -133,8 +133,8 @@ func (f *fakeHTTP) Get(url string) (*http.Response, error) {
 }
 
 func TestRunHandlerHttp(t *testing.T) {
-	fakeHttp := fakeHTTP{}
-	handlerRunner := NewHandlerRunner(&fakeHttp, &fakeContainerCommandRunner{}, nil)
+	fakeHTTPGetter := fakeHTTP{}
```

Comment:
Thanks for the catch but could we remove this package from the go lint files?

Reply:
OK, I see the comments above

Reply:
I only fix one file and there are many golint errors in this package.